README.md

```diff
@@ -11,3 +11,20 @@ Watch the full companion video on YouTube:
 
 ## [Start the hands-on here >>](https://dynatrace.github.io/obslab-predictive-kubernetes-scaling)
 
+### 2.Create a GKE cluster
```

Comment:
This step is optional right? Could we please add a bit more information to the documentation? It would be great to mention that there are two paths to follow, why it is like that and which path is recommended in which situation and so on.